src/components/search/Search.js

```diff
@@ -41,7 +41,7 @@ class Search extends Component {
             type: 'text',
             ref: 'searchInput',
             className: 'input-search',
-            placeholder: 'Just type a song name and press Enter'
+            placeholder: 'Type a song name and select an option'
```

Comment:
Yep. Didn't want to open a new branch for this. Changes in app functionality where made, and nobody checked with the copywritting team.
It's an UX matter anyway. If we change the title from "Responsive" to "UX Issues" we are ok. :P